### DISCUSSION TITLE
React native Evastore template Not working with Latest Expo 46

### DISCUSSION THREAD

**Discussion by BhaveshSarwar** [inspireui/support#5617] [New Feature/Ideas Requests]
Hello we have bought your plugin for Evastore shopify react native app
It uses expo version 36.0 and currently expo is running on 46.0 version
Due to which are not able to upload builds 
as code generation for native is quite difficult because of very old plugin and libraries
Updating all plugin is quite tedious task and we want to release the build asap
We are stucked because apple does not allow to uploade build below target 14.2 compiled and android does not allow below api level 30 
Please help us to update this code 

https://codecanyon.net/item/evastore-shopify-mobile-store-by-react-native-and-graphql/22715163#

**Comment by snabbastAdmin** [comment]
@minhcasi @luyx2412 
We are also stuck with the same problem, Kindly look into this, we are stuck with the release. This is really urgent for us.

**Comment by minhcasi** [comment]
@BhaveshSarwar @snabbastAdmin - Could you help to share the screenshot/videos of the issues and post them to our Ticket Support website - https://support.inspireui.com/ 

Our technical team will help with supporting and following up on the issue. 

Sincerely
